nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2100,7 +2100,7 @@ this property specifies the maximum amount of time to keep the archived data. It
 
 === Provenance Repository
 
-The Provenance Repository contains the information related to Data Provenance. The next three sections are for Provenance Repository properties.
+The Provenance Repository contains the information related to Data Provenance. The next four sections are for Provenance Repository properties.
 
 |====
 |*Property*|*Description*
```

Comment:
should add formatting to the the values in this first paragraph for consistency ie. (`org.apache.nifi.provenance.PersistentProvenanceRepository` and `org.apache.nifi.provenance.VolatileProvenanceRepository`)